src/wallet/spend.cpp

```diff
@@ -1091,15 +1091,14 @@ static util::Result<CreatedTransactionResult> CreateTransactionInternal(
     txNew.vout.reserve(vecSend.size() + 1); // accommodate the possible later inserts
     for (const auto& recipient : vecSend)
     {
-        CTxOut txout(recipient.nAmount, GetScriptForDestination(recipient.dest));
+        auto& txout = txNew.vout.emplace_back(recipient.nAmount, GetScriptForDestination(recipient.dest));
```

Comment:
One last nit: because it's not modified after creation, `const auto& txout`

Reply:
Done, thanks